src/test/java/com/remondis/remap/MapperTest.java

```diff
@@ -323,7 +323,7 @@ public void shouldMapToNewSet() {
   public void shouldDenyIllegalArguments() {
 
     assertThatThrownBy(() -> {
-      Mapping.from(null);
+      Mapping.from((Class<?>)null);

```

Comment:
This should be `Mapping.from((GenericTypeReference<?>) null);` to check the right method for `null`.